src/ts/client/grapesjs/keymaps.ts

```diff
@@ -0,0 +1,64 @@
+import {Editor, PluginOptions} from 'grapesjs'
+import {cmdPublish} from './PublicationUi'
+import {cmdOpenFonts} from '@silexlabs/grapesjs-fonts'
+import {cmdToggleBlocks, cmdToggleLayers, cmdToggleNotifications, cmdToggleSymbols} from './index'
+import {cmdTogglePages} from './page-panel'
+import {cmdOpenSettings} from './settings'
+
+// Utility functions
+
+const selectBody = (editor: Editor): void => {
```

Comment:
I'd say this should be made a command and it should probably be defined in utils or somewhere where it belongs

Reply:
I agree!

Reply:
It has been! When I update the branch, a command declaration will be written in ``keymaps.ts`` and the selectBody() function will be in ``utils.ts``